frontend/simple/views/containers/Proposals.vue

```diff
@@ -1,42 +1,24 @@
 <template>
   <div>
-    <!--  Start Original Voting Banner to delete soon -->
-    <voting-banner v-for="proposal in Object.values(proposals)"
-      :proposal="proposal"
-    />
-    <!--  End Original Voting Banner to delete soon -->
-
-    <!-- REVIEW: Not sure about these banners. -->
-
-    <!--
-    <h4 class="title is-size-5 notification is-warning gi-is-banner gi-notify"
-      v-if="groupProposals.notVoted.length"
-    >
-      <i18n>These are waiting for your vote!</i18n>
-    </h4>
-    -->
 
     <i18n class="notification gi-is-banner gi-notify" v-if="allVoted">Cool, you already voted on all proposals.</i18n>
 
     <voting
-      v-for="proposal in groupProposals.notVoted"
-      :type="proposal.type"
-      :proposal="proposal.data"
+      v-for="proposal in proposalList.notVoted"
```

Comment:
At GroupDashboard.vue I'd only show `Proposals.vue` if there are any open proposals, otherwise the users see's an section with a title without any content.

Reply:
If there are no proposals there should also be no "Proposals" section visible in the Dashboard

Reply:
That's exactly what I've meant . What did you understood? xD